Apps/W1/Shopify/app/src/Payments/Pages/ShpfyPaymentTermsMapping.Page.al

```diff
@@ -0,0 +1,71 @@
+namespace Microsoft.Integration.Shopify;
+
+/// <summary>
+/// Page Shpfy Payment Terms Mapping (ID 30162).
+/// </summary>
+page 30162 "Shpfy Payment Terms Mapping"
+{
+    PageType = List;
```

Comment:
I'm not sure, do they mark pages as internal? if so, we should do it as well

Reply:
They don't mark pages as internal, so I also won't do that